src/NLog/Targets/Target.cs

```diff
@@ -417,7 +417,9 @@ internal void Initialize(LoggingConfiguration configuration)
 
                 if (!IsInitialized)
                 {
-                    PropertyHelper.CheckRequiredParameters(this);
+                    if (configuration == null)
```

Comment:
Why do we need this here? We have already checked IsInitialized

Reply:
Because some unit-tests still create and use targets without using the front-door. Notice how the code-coverage is all green.